pkg/sdk/python/Makefile

```diff
@@ -0,0 +1,17 @@
+default: all
+.PHONY: clean all default
+# TODO(fred): version lib
+all: datamon.so
+# Define this in build environment (2.7 is default)
```

Comment:
data science has upgraded to python3, so i suggest we default to 3 as well if/when it's time to move this beyond PoC, ideally 3.6+ in order to allow some recent-ish niceties available in Python's `asyncio` stdlib.

Reply:
yeah it also works with python3, but this needs a different build